x-pack/plugins/security/public/management/roles/edit_role/edit_role_page.tsx

```diff
@@ -526,7 +526,7 @@ export const EditRolePage: FunctionComponent<Props> = ({
       try {
         await rolesAPIClient.saveRole({ role, spacesEnabled: spaces.enabled });
       } catch (error) {
-        notifications.toasts.addDanger(get(error, 'data.message'));
+        notifications.toasts.addDanger(get(error, 'body.message'));
```

Comment:
Nothing wrong with your approach, but now that we have support for nullish coalescing and optional chaining, we can remove our usage of `lodash.get`. I've also suggested a fallback error message in the event that `error.body.message` isn't defined:


```suggestion
        notifications.toasts.addDanger(
          error?.body?.message ??
            i18n.translate('xpack.security.management.editRole.errorSavingRoleError', {
              defaultMessage: 'Error saving role',
            })
        );
```

Reply:
@legrego Thank you for reviewing. I have addressed this and the other comment about deleteRole in the recent commit.